remove-null-values/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Practice problems remove-null-values</title>
+  </head>
+  <body>
+
+
+    <script>
+
+
+      function removeNullValues(array) {
+        var newArray = [];
+        var oldArray = [];
+        for (var i = 0; i < array.length; i++) {
+          if (array[i]) {
```

Comment:
This is a good starting point and will remove all null values, but it will also remove all "falsy" values. Make sure to update it!